File: src/sdk-reference/js/6/realtime/count/snippets/count.js
function callback (notification) {}

try {
  const roomId = await kuzzle.realtime.subscribe('nyc-open-data', 'yellow-taxi', {}, callback);

  const count = await kuzzle.realtime.count(roomId);

  console.log(`Currently ${count} active subscription`);
} catch (error) {
  console.error(error.message);
}

File: src/sdk/java/1/controllers/realtime/count/snippets/count.java
NotificationListener listener = new NotificationListener() {
  public void onMessage(NotificationResult notification) {}
};

try {
    String roomId = kuzzle.getRealtime().subscribe(
      "nyc-open-data", 
      "yellow-taxi", 
      "{}", 
      listener
    );

    int count = kuzzle.getRealtime().count(roomId);

    System.out.println("Currently " + count + " active subscription");
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}
